src/ngraph/runtime/cpu/pass/cpu_assignment.cpp

```diff
@@ -527,6 +528,21 @@ namespace ngraph
                     update_slice->set_op_annotations(op_annotations);
                 }
 
+                template <>
+                void CPUAssignment::ASSIGN_DECL(ngraph::op::ScatterAdd)
+                {
+                    auto update_slice = static_cast<ngraph::op::ScatterAdd*>(node);
+
+                    auto op_annotations =
+                        std::make_shared<ngraph::runtime::cpu::CPUOpAnnotations>();
+                    if (get_user_count(node->get_argument(0).get()) == 1)
+                    {
+                        // Safe to overwrite input
```

Comment:
unittest for this ?

Reply:
Added two unit test for this. There were 3 unit tests for ScatterAdd before this PR: scatter_add_2d_indices, scatter_add_1d_indices, and scatter_add_scalar_indices.